<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>VoipMainDialog</class>
 <widget class="QDialog" name="VoipMainDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>480</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>可视对讲</string>
  </property>
  <widget class="CommonPushButton" name="pushButtonVoipMainDial">
   <property name="geometry">
    <rect>
     <x>610</x>
     <y>320</y>
     <width>170</width>
     <height>60</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>AlArabiya</family>
     <pointsize>12</pointsize>
    </font>
   </property>
   <property name="focusPolicy">
    <enum>Qt::NoFocus</enum>
   </property>
   <property name="text">
    <string>拨号</string>
   </property>
  </widget>
  <widget class="CommonPushButton" name="pushButtonVoipMainMon">
   <property name="geometry">
    <rect>
     <x>610</x>
     <y>260</y>
     <width>170</width>
     <height>60</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>AlArabiya</family>
     <pointsize>12</pointsize>
    </font>
   </property>
   <property name="focusPolicy">
    <enum>Qt::NoFocus</enum>
   </property>
   <property name="text">
    <string>视频 </string>
   </property>
  </widget>
  <widget class="CommonPushButton" name="pushButtonVoipMainReturn">
   <property name="geometry">
    <rect>
     <x>610</x>
     <y>380</y>
     <width>170</width>
     <height>60</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>AlArabiya</family>
     <pointsize>12</pointsize>
    </font>
   </property>
   <property name="focusPolicy">
    <enum>Qt::NoFocus</enum>
   </property>
   <property name="text">
    <string>返回</string>
   </property>
  </widget>
 </widget>
 <customwidgets>
  <customwidget>
   <class>CommonPushButton</class>
   <extends>QPushButton</extends>
   <header>commonpushbutton.h</header>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
